src/gui/fspathedit.h

```diff
@@ -105,7 +105,7 @@ protected slots:
     Q_DISABLE_COPY(FileSystemPathEdit)
     class FileSystemPathEditPrivate;
     Q_DECLARE_PRIVATE(FileSystemPathEdit)
-    QScopedPointer<FileSystemPathEditPrivate> const d_ptr;
+    const QScopedPointer<FileSystemPathEditPrivate> d_ptr;
```

Comment:
@Chocobo1:  Did you intend to leave this line as `QScopedPointer`?

Reply:
Yes, older Qt (on our CI) seems to have issues with std::unique_ptr here.

ps. Please don't delete your comments, it might be useful for others.

Reply:
Here is overusing of QScopedPointer.
I would just have raw pointer and delete it in destructor. What's the problem?

Reply:
https://ci.appveyor.com/project/qbittorrent/qbittorrent/builds/22417117#L521

Reply:
>What's the problem?

This question was not about the problem with unique_ptr. It was "What's the problem with raw pointer here?". There's no reason to use any kind of smart pointer in this case.

Reply:
>What's the problem with raw pointer here?

As the original author is absent, you probably won't get a genuine answer.

Reply:
Ah, the original author... I'm afraid I know what answer I'll get from him. I wanted other opinions.

Reply:
>I would just have raw pointer and delete it in destructor.

I'm ok with it.

Reply:
Note for myself
>Yes, older Qt (on our CI) seems to have issues with std::unique_ptr here.

https://github.com/qt/qtbase/commit/98ef4239a6b859522a45bdd03ba5496ea201e5f3

Reply:
👍 
In which Qt version is it changed?

Reply:
>In which Qt version is it changed?

Looks like changed since 5.12.